src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  return (firstInput) => {
+    return (secondInput) => {
+      callCount++;
+
+      if (callCount <= 3) {
+        return firstInput + secondInput;
+      }
+
+      if (callCount % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return firstInput + secondInput;
```

Comment:
You have two options with same return. Think about how you can simplify conditions.

Reply:
Agree. Done ))